Jenkinsfile

```diff
@@ -142,15 +156,12 @@ pipeline {
 
                     // Install additional tools:
                     // - ssh client for govm
-                    // - sudo for non-privileged user
-                    sh "docker exec builder swupd bundle-add openssh-client sudo"
+                    sh "docker exec builder swupd bundle-add openssh-client"
 
-                    // Allow non-root users to become root via sudo when they are part of the root group.
-                    // ${DockerBuildArgs()} includes the necessary parameters for that.
-                    sh "docker exec builder mkdir /etc/sudoers.d"
-                    sh "echo '%root ALL=(ALL) NOPASSWD: ALL' | docker exec -i builder tee /etc/sudoers.d/nopasswd >/dev/null"
+                    // Make /usr/local/bin writable for all users. Used to install kubectl.
+                    sh "docker exec builder sh -c 'mkdir -p /usr/local/bin && chmod a+wx /usr/local/bin'"
 
-                    // sudo needs a user entry for the jenkins user.
+                    // Some tools expect a user entry for the jenkins user (like govm?)
```

Comment:
not strictly in scope of this change, but changed comment directed my attention to following two lines. Isnt creation of lines with hard-coded values and number of fields directly in /etc/passwd and /etc/shadow brutal and non-portable way? There is portable and cleaner method called useradd, is there reason we can not use it?

Reply:
Yes, it might be cleaner, but the current approach worked and I didn't want to invest further time into making it better.

Regarding "portable": this isn't that important, because neither the OS we use this on nor the format of `/etc/passwd` or `/etc/group` are likely to change.